src/server/models/app-models.js

```diff
@@ -305,6 +372,26 @@ export default function (db) {
     }
   }
 
+  AuditEvent.belongsTo(AuditClientData, {
+    as: 'clientData',
+    foreignKey: 'clientDataId',
+    targetKey: 'id',
+  })
+
+  AuditClientData.hasMany(AuditEvent, {
+    as: 'clientData',
+    foreignKey: 'clientDataId',
+    sourceKey: 'id',
+  })
+
+  AuditEvent.belongsTo(User, {
+    onDelete: 'CASCADE',
```

Comment:
We definitely do not want the user deletions to cause deletions of audit events, or vice versa. The audit events should live essentially forever, regardless of any data. Usually there wouldn't even be these foreign key relationships on the db level, but we can live with them. We just can't have the events being removed because a user might

Reply:
Does sequelize support relations without creating a foreign key? That would make things much easier.

Reply:
It does and I changed it to that. Now there are no foreign keys and if an user gets deleted for some reason it doesn't affect the audit_events table at all and the frontend just says "unknown user".